From 9a696a81918341806941e57005fa9533486c469a Mon Sep 17 00:00:00 2001
From: Stefan Chulski <stefanc@marvell.com>
Date: Wed, 20 Mar 2019 12:52:20 +0200
Subject: [PATCH 064/386] arm64: dts: a8040: enable USB2 on CP0 and CP1 for
 setup H

Change-Id: I821f3c724886c586f016dad2fb561cce8264e90e
Signed-off-by: Stefan Chulski <stefanc@marvell.com>
Reviewed-on: https://sj1git1.cavium.com/5991
Tested-by: sa_ip-sw-jenkins
Reviewed-by: Stefan Chulski <Stefan.Chulski@cavium.com>
[RH: Original patch taken from marvell 88F3720 board support SDK 10.0-PR2003]
Signed-off-by: Ruiqiang Hao <Ruiqiang.Hao@windriver.com>
---
 .../boot/dts/marvell/armada-8040-db-H.dts     | 24 +++++++++++++++++--
 1 file changed, 22 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/boot/dts/marvell/armada-8040-db-H.dts b/arch/arm64/boot/dts/marvell/armada-8040-db-H.dts
index 44c86b7c0cc8..4c8ef568a69e 100644
--- a/arch/arm64/boot/dts/marvell/armada-8040-db-H.dts
+++ b/arch/arm64/boot/dts/marvell/armada-8040-db-H.dts
@@ -85,6 +85,20 @@
 	phy-mode = "rgmii-id";
 };
 
+&cp0_crypto {
+	status = "okay";
+};
+
+/* CON9 on CP0 expansion */
+&cp0_usb3_0 {
+	status = "okay";
+};
+
+/* CON10 on CP0 expansion */
+&cp0_usb3_1 {
+	status = "okay";
+};
+
 &cp1_spi1 {
 	status = "okay";
 };
@@ -128,10 +142,16 @@
 	phy-mode = "rgmii-id";
 };
 
-&cp0_crypto {
+&cp1_crypto {
 	status = "okay";
 };
 
-&cp1_crypto {
+/* CON9 on CP1 expansion */
+&cp1_usb3_0 {
+	status = "okay";
+};
+
+/* CON10 on CP1 expansion */
+&cp1_usb3_1 {
 	status = "okay";
 };
-- 
2.17.1

